src/file-upload/styles.scss

```diff
@@ -9,3 +9,7 @@
 .hints {
   /* used in test-utils */
 }
```

Comment:
This is now unnecessary

Reply:
It is not but I like mentioning it still. Like in other file I mention more class names:

```
.file-option-name,
.file-option-size,
.file-option-last-modified,
.file-option-thumbnail {
  /* used in test-utils */
}
```

It does not matter if there are styles defined for these or not, by looking at the file's header you know which classes are used as test selectors.

Reply:
I can accept this explanation of all utils are grouped together. But when there are only two selectors `.root, .hints` it is not useful